pkg/controller/migmigration/restore.go

```diff
@@ -258,13 +285,16 @@ func (t *Task) hasRestoreCompleted(restore *velero.Restore) (bool, []string) {
 		progress = append(progress, stageReport...)
 	case velero.RestorePhaseCompleted:
 		completed = true
+		itemsRestored, totalItems := getRestoreStats(restore)
 		progress = append(
 			progress,
 			fmt.Sprintf(
-				"Restore %s/%s: %s",
+				"Restore %s/%s: %d out of estimated total of %d objects restored%s",
```

Comment:
A minor readability point: It might be better to remove the initial space from the duration string and put it between "restored" and "%s" above.